spec/rubocop/cop/rails/inverse_of_spec.rb

```diff
@@ -142,4 +142,92 @@ class Picture < ApplicationRecord
       RUBY
     end
   end
+
+  context 'with valid options in `with_options`' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<-RUBY.strip_indent)
+      class Person
```

Comment:
I think that it is easier to read indents after heredoc.

```ruby
expect_no_offenses(<<-RUBY.strip_indent)
  class Person
    ...
  end
RUBY
```

Reply:
👍

Reply:
I think so, too. I will also fix indents of other examples.